### DISCUSSION TITLE
[Feature Request]: Using CRDT Libraries to implement conflict resolution

### DISCUSSION THREAD

**Discussion by theMasix** [remotely-save/remotely-save#455] [Ideas]
### What feature are you suggesting?

There is an amazing project called CRDT that helps handle conflict resolution problems.
https://crdt.tech/

Also, there are some implementations of its algorithm:
https://crdt.tech/implementations

Any idea about using these libraries and integrating them with the plugin?

### What remote cloud services are you using, or suggesting adding the feature to?

S3

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
the most difficulty comes from the fact that the plugin assumes only dummy server (s3 as storage). i will think about it

**Comment by theMasix** [comment]
@fyears yeah. the plugin should look at the current page as well.
But CRDT handles the most difficult part, the algorithm!
